client/src/styles/textcheckboxgroup.scss

```diff
@@ -1,4 +1,9 @@
-.textcheckboxgroup .input-group-append .form-check-input {
-  margin-left: 0;
-  position: relative;
+.textcheckboxgroup {
+  .form-check-input {
+    margin-left: 0;
+    position: relative;
+  }
+  .form-label {
+    margin-bottom: 0;
+  }
```

Comment:
Counteracts bootstrap adding a margin for this which broke the "display title?" checkbox styling.